### DISCUSSION TITLE
Certification Link for Our Privacy Opportunity

### DISCUSSION THREAD

**Discussion by Profpwner** [OpenMined/courses#324] [Course: Our Privacy Opportunity]
Hi,

I passed the final project for "Our Privacy Opportunity". It said there was a certification link but I did not see it:

![image](https://user-images.githubusercontent.com/15971116/116822193-53592e00-ab32-11eb-86a7-f761263eb90a.png)

I was wondering how I could get the certification.

Regards,
Abe

**Comment by znreza** [comment]
Hi @Profpwner the team is working on the certificates. Once they are ready to distribute, you will be notified I believe! Hope you liked the course.

**Reply by Profpwner** [reply]
The course was awesome!!!  Thanks so much for putting these classes together.

**Reply by pjoshireply** [reply]
Hi @znreza any updates on the certificates? I have been waiting for some updates on this front too :)

**Comment by cossiomanuel** [comment]
Hi guys,
The course was great! Very well designed! I was wondering if you advanced on the topic of the certificates. I continue to wait for mine!

Thank you very much for the help.

Manuel

**Comment by anilpatil-securityarchitect** [comment]
share the sample of certificate with community. I was waiting for this certificate since 2 years. I search certificate sample or anyone finished this course post on linkedin but not succeed .
